src/Lib/Canvas.vala

```diff
@@ -87,6 +87,22 @@ public class Akira.Lib.Canvas : Goo.Canvas {
         window.event_bus.set_focus_on_canvas.connect (on_set_focus_on_canvas);
     }
 
+    public void insert_item_default (Akira.Lib.Models.CanvasItem item, bool select = false) {
+        items_manager.add_item (item);
+        // TODO: place in the viewport
+        Utils.AffineTransform.set_position (
+            Akira.Layouts.MainCanvas.CANVAS_SIZE / 2,
+            Akira.Layouts.MainCanvas.CANVAS_SIZE / 2,
+            item    
+        );
+
+        if (select) {
```

Comment:
Is there any scenario where we don't want to select the last inserted image?

Reply:
In the code, no, but let's say you import a lot of images and the app should (?) just select the last one.

Reply:
Yeah, I think only the last one should be selected, at least for now.
Once we implement the multi-selection option we will update this.
For now, it'd be great to have `select` always `false` unless is the last uploaded image.